config.yaml

```diff
@@ -0,0 +1,88 @@
+# Main gamestate and step to load
```

Comment:
Shouldn't this not be in source control. The reason to have something like this is so that each person could have their own config and not have to worry about stashing and stuff. Maybe call this `config.yaml.example` and then add `config.yaml` to `.gitignore`.

Reply:
Agreed. I had to add the example yaml somewhere, or everything would break for everyone. Ideally as is written in a comment somewhere, it should maybe be possible to open an arbitrary file rather just specifically `config.yaml`. More opinions needed on how people want it to be done.